src/runtime/RegExpObject.cpp

```diff
@@ -457,6 +458,18 @@ void RegExpObject::createRegexMatchResult(ExecutionState& state, String* str, Re
     } while (testResult);
 }
 
+static void invalidateLegacyRegExpStaticProperties(ExecutionState& state)
+{
+    state.resolveCallee()->codeBlock()->context()->regexpStatus().input = String::emptyString;
+    state.resolveCallee()->codeBlock()->context()->regexpStatus().lastMatch = StringView();
+    state.resolveCallee()->codeBlock()->context()->regexpStatus().lastParen = StringView();
+    state.resolveCallee()->codeBlock()->context()->regexpStatus().leftContext = StringView();
+    state.resolveCallee()->codeBlock()->context()->regexpStatus().rightContext = StringView();
```

Comment:
It seems like that `RegExpStatus` could be accessed simply by `state.context()->regexpStatus()`